tests/test_scripts.py

```diff
@@ -1079,66 +1075,39 @@ def _make_graph(infilename, min_hashsize=1e7, n_hashes=2, ksize=20,
     return outfile
 
 
-def _DEBUG_make_graph(infilename, min_hashsize=1e7, n_hashes=2, ksize=20,
```

Comment:
It is unused; `ipdb` provides a better debugging interface (just drop `import ipdb; ipdb.set_trace()` into the code in question) without mucking up the code.